examples/pytorch/ogb/ngnn/seal/main.py

```diff
@@ -282,7 +282,8 @@ def print_log(*x, sep="\n", end="\n", mode="a"):
         type=str,
         default="none",
         choices=["none", "input", "hidden", "output", "all"],
-        help="You can set this value from 'none', 'input', 'hidden' or 'all' to apply NGNN to different GNN layers.",
+        help="You can set this value from 'none', 'input', 'hidden' or 'all' "
```

Comment:
help="You can set this value from 'none', 'input', 'hidden' or 'all' " \
          "to apply NGNN to different GNN layers.",